package.json

```diff
@@ -19,10 +19,11 @@
     "build-server": "webpack -p --config build/webpack-server-conf.js --progress"
   },
   "dependencies": {
-    "@creativecommons/vue-vocabulary": "0.0.0-alpha.1",
+    "@creativecommons/vocabulary": "1.0.0-beta.2",
     "@sentry/browser": "^5.6.0",
     "@sentry/node": "^5.6.0",
     "axios": "^0.18.0",
+    "bulma": "^0.8.0",
```

Comment:
I thought we weren't going to add bulma directly and only use specific classes via Vocabulary?

Reply:
We need to install bulma if we want to import Vocabulary's scss files, because they import Bulma's styles. If Bulma isn't inside `node_modules`, it doesn't build